docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,156 @@
+
+=== New Task List and Task Administration perspectives
+The former Task List perspective has been divided into the Tasks List and Task Administration perspectives. The Task List perspective is used by task operators to work with their assigned (or potentially assigned) tasks. The Task Administration perspective is used by administrators to manage tasks assigned to other users. This perspective is only available for users with admin or process-admin roles. It is similar to the Admin filter tab on the former Task List perspective.
+
+=== Improved validation in deployment descriptor editor
+
+The deployment descriptor editor in Business Central has been improved to validate data entered for the following:
+
+* Work item handlers
+* Event listeners
+* Marshalling strategies
+* Globals
+* Environment entries
+* Configuration
+
+Validation is performed automatically when a build is invoked or manually when a user clicks *Validate*.
+
+=== Error Handling
+The system now detects and stores execution errors such as exceptions while running a process instance or a job after the maximum number of retries has been exceeded. If an error occurs, an event is generated that contains the following data:
+
+--
+[cols="1,2",options="header"]
+|===
+| Data
+| Fields
+
+.3+| Process model
+| processModelID
+| processModelVersion
+| processModelName
+
+.2+| Process instance that has failed
+| processInstanceID
+| customProcessID
+
+.5+| Activity that has failed
+| activityID
+| activityName
+| activityType
+| iteration
+| status
+
+.2+| Error data
+| errorMessage: Message with the error that has occurred.
+| errorStack: Stack with error detail.
+|===
+--
+
+The process instance list includes a new column that displays the number of errors per process instance which are not marked as acknowledged. When you select the column, a window appears which will enables you to navigate to the new Error perspective.
+
+The Execution Errors perspective lists any errors that occur during process instances or executor service jobs on a Decision Server. Use this perspective to help troubleshoot issues. It is visible to users with the admin and process-admin roles.
+
+
+== Process Engine
+
+=== Process and task administration API
+A process administration API and task administration API have been introduced to simplify some of the more complex administrator use cases.
+The process admin API enables you to:
+
+* Retrieve all process definition nodes
+* Cancel node instances
+* Retrigger node instances
+* Update the timer (absolute or relative)
+* List timer instances
+* Trigger nodes
+
+The task administration API enables you to:
+
+* Add and remove potential owners and excluded owners and business administrators
```

Comment:
So you can use the API to add and remove potential owners, excluded owners, but not actual assigned owners? Are you sure?